certify-service/src/main/resources/application-local.properties

```diff
@@ -27,7 +27,7 @@ mosip.certify.authn.filter-urls={ '${server.servlet.path}/issuance/credential',
 mosip.certify.authn.issuer-uri=http://localhost:8088/v1/esignet
 mosip.certify.authn.jwk-set-uri=http://localhost:8088/v1/esignet/oauth/.well-known/jwks.json
 mosip.certify.authn.allowed-audiences={ '${mosipbox.public.url}${server.servlet.path}/issuance/credential', 'http://localhost:8088/v1/esignet/vci/credential' }
-
+mosip.certify.well-known.file.uri=https://raw.githubusercontent.com/mosip/inji-config/develop/certify-mock-openid4vci-wk.json
```

Comment:
We should rename this file to certify-mock-identity-issuer-metadata.json.
Config files for other usecases should be renamed to match this naming convention.

Reply:
In the previous review Anusha had suggested to specify the Spec for well-known as well, i.e. `openid4vci` as there're other well-knowns as well.

ref: https://github.com/mosip/inji-certify/pull/49#discussion_r1682618183

How about renaming it to `certify-mock-identity-issuer-openid4vci-well-known.json` ?

How many other well-known configurations do we expect to see in the Inji Stack/MOSIP Stack in the near future?

Reply:
It will not be immediate, so I suggest leaving the openid4vci part since it will make the file name too big.

Reply:
I actually mean to release the openid4vci from the name, but I think the file name is change to just openid4vci